newswires/app/conf/SearchPresets.scala

```diff
@@ -157,14 +158,227 @@ object SearchPresets {
     )
   )
 
+  private val ReutersBusiness = List(
+    SearchParams(
+      text = None,
+      suppliersIncl = List("REUTERS"),
+      categoryCodesIncl = List(
+        "REUTERS:RBN"
```

Comment:
The `RBN` destination tag looks like it's on the majority of non-sport Reuters stories, so including it feels overly broad for Business to me.. e.g. a number of stories about the Pope election and Swedish Prime Minister are present in the feed at the moment due to this tag.

The name 'Reuters Business News' makes it sound like it _should_ be ideal here though! Have we cross-checked to see whether these stories are also showing up in the existing Fingerpost business lists?